src/load-data.js

```diff
@@ -1,5 +1,34 @@
 'use strict'
 
+const TILL_KEY = 'SPLITIO.splits.till'
+
 export default function loadDataIntoLocalStorage ({ serializedData = {} }) {
-  throw new Error('Not implemented')
+  const { segmentsData, since, splitsData, userId, usingSegmentsCount } = serializedData
+  const windowLocalStorage = window.localStorage
```

Comment:
Would it be useful if this was an argument to the function (the a default value of `window.localStorage`)? That might, for example, simplify testing (you can use dependency injection instead of jsdom mocking).